homeassistant/components/upb/light.py

```diff
@@ -57,7 +57,7 @@ class UpbLight(UpbAttachedEntity, LightEntity):
     def __init__(self, element, unique_id, upb):
         """Initialize an UpbLight."""
         super().__init__(element, unique_id, upb)
-        self._brightness = self._element.status
+        self._attr_brightness: int = self._element.status
```

Comment:
Not sure it is allowed to set `brightness` to `0`
See: https://developers.home-assistant.io/docs/core/entity/light/

Reply:
Good point, but must say that this was like this before as well so it doesn't fix it but it also doesn't make it worse

Reply:
May be you could add `| None`

Reply:
But that would mean a logic change as we would check if it's 0, we return None. This PR was intended not to change any logic. I also don't know if the change of transforming 0 to None is changing any behaviour